readme.md

```diff
@@ -4,6 +4,8 @@
 
 ![](screenshot.png)
 
+**Note:** Looking for a version compatible with Ink 2.x? Check out [previous release](https://github.com/sindresorhus/ink-gradient/tree/v1.0.0).
```

Comment:
```suggestion
**Note:** Looking for a version compatible with Ink 2.x? Check out [this release](https://github.com/sindresorhus/ink-gradient/tree/v1.0.0).
```

It will technically no longer be the previous release when I do a v3.0.1, for example.